darshan-util/pydarshan/docs/install.rst

```diff
@@ -1,53 +1,67 @@
 .. highlight:: shell
 
+.. _installation:
+
 ============
 Installation
 ============
 
 
-Stable release
---------------
+Stable release (PyPI)
+---------------------
 
-To install PyDarshan, run this command in your terminal:
+The preferred method for installing PyDarshan is to use `pip`_ to install stable
+releases directly from PyPI:
 
 .. code-block:: console
 
     $ pip install darshan
 
-This is the preferred method to install PyDarshan, as it will always install the most recent stable release.
-If you don't have `pip`_ installed, this `Python installation guide`_ can guide
-you through the process.
+PyDarshan releases on PyPI include manylinux and macOS wheels that simplify
+distribution of PyDarshan and its dependencies for most users.
```

Comment:
It may be worth providing a bit more detail here--many of the core Python projects now provide binaries for ARM-based architectures (Linux and Mac), but we don't do that yet. So, perhaps just note that we are x86 only for binaries at the moment?

This would be pretty easy to change in the future with `cibuildwheel`.